experimental/dds/tree2/src/feature-libraries/schemaSummarizer.ts

```diff
@@ -43,18 +39,20 @@ const schemaStringKey = "SchemaString";
 export class SchemaSummarizer implements Summarizable {
 	public readonly key = "Schema";
 
-	private schemaBlobCache: IFluidHandle<ArrayBufferLike> | undefined;
 	private readonly codec: IJsonCodec<TreeStoredSchema, Format>;
 
+	private schemaIndexLastChangedSeq: number | undefined;
+
 	public constructor(
 		private readonly runtime: IFluidDataStoreRuntime,
 		private readonly schema: StoredSchemaRepository,
 		options: ICodecOptions,
+		collabWindow: { getCurrentSeq(): number | undefined },
```

Comment:
you should make a named interface for this with documented semantics. Suggestion to make it align with merge-tree is to call it `CollaborationWindow` (`CollabWindow` is ok too)

Reply:
Added interface `CollabWindow` in `schemaSummarizer`